src/transformState.js

```diff
@@ -46,7 +46,23 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const obj in transforms) {
+    const currentObj = transforms[obj];
```

Comment:
at this moment you got in your obj indexes of the array, use "for of" loop for arrays

Reply:
transforms is an object, that why I use for...in loop